### ISSUE TITLE
Template error is lacking some detail

### ISSUE THREAD

**Issue by benjaminapetersen** [mozilla/nunjucks#1095]
Pretty simple `express` + `nunjucks` setup, code is:

```javascript 
app.get('/', (req, res) => {
  res.render(`views/index.html`, {
    title: 'Index.html',
    items: [
      {name: 'item 1'},
      {name: 'item 2'},
      {name: 'item 3'}
    ]
  });
});
```

```
Error: template not found: err.nunjucks
    at createTemplate (my_proj/node_modules/nunjucks/src/environment.js:274:15)
    at next (my_proj/node_modules/nunjucks/src/lib.js:283:7)
    at handle (my_proj/node_modules/nunjucks/src/environment.js:313:11)
    at my_proj/node_modules/nunjucks/src/environment.js:323:9
    at next (my_proj/node_modules/nunjucks/src/lib.js:281:7)
    at Object.asyncIter (my_proj/node_modules/nunjucks/src/lib.js:287:3)
    at Environment.getTemplate (my_proj/node_modules/nunjucks/src/environment.js:305:9)
    at Environment.render (my_proj/node_modules/nunjucks/src/environment.js:344:10)
    at NunjucksView.render (my_proj/node_modules/nunjucks/src/express-app.js:20:9)
    at tryRender (my_proj/node_modules/express/lib/application.js:640:10)
```

File is there. was sent as a static html file before adding `nunjucks` to the project.  The error is pretty cryptic, should it tell me what template it isn't finding?

**Comment by Spown** [comment]
I think it's all a part of a larger problem that was introduced some versions ago, where some error details are being lost / are not getting to output... Probably related to #1087

**Comment by fdintino** [comment]
This is now fixed in the 3.1.3 release
